docs/source/workflow.rst

```diff
@@ -193,3 +193,51 @@ monitoring checks for free:
   instances are deployed on a cluster. If the service is registered in Smartstack
   it will look in Smartstack to count the available instances. Otherwise it
   counts the number of healthy tasks in Mesos.
+
+
+The PaaSTA command line
+------------------------
+
+The PaaSTA command line interface, ``paasta``, gives users of PaaSTA the
+ability to inspect the state of services, as well as stop and start existing
+services. See the man pages for a description and detail of options for any
+individual paasta command.  Some of the most frequently used commands are
+listed below:
+
+   * ``paasta start`` - sets the desired state of the service instance to
```

Comment:
This is good, but I wish we didn't have to duplicate it.

This "could" be the docstring of start/stop, and then the help text could just read the help string as part of the argparser, which would in turn get sucked into the manpage.
```
   add_parser(
     epilog=help(paasta_start)

def paasta_start()
"""Sets the desired state of the service....."""
```


Then in sphinx could we do
http://www.sphinx-doc.org/en/stable/domains.html#cross-referencing-python-objects

And do like...
```
See the man pages for fequlently bla:

  * ``paasta start`` - :py:meth:help(paasta_tools.cli.paasta_start)
```

This is all mostly a wishlist here. 

I won't block a shipit on being fancy with sphinx, but I will ask that you copy these docs into the help text of the commands themselves so they show up with --help (and end up in the manpage)